### PR TITLE
Updates requests and urllib3 requirements

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
# Description

Updates urllib3 dependency from 1.25.8 to 1.25.10 (and requests from 2.20.0 to 2.22.0) to pass safety check.

# Test Plan

- [ ] does CI pass?

### CODE REVIEW

**Review by emkll — CHANGES_REQUESTED** [review]
`build-requirements.txt` is in the `requirements/` directory and not the repo root for this repo.

**Comment by emkll** [comment]
Rebased on latest main

**Review by creviera — CHANGES_REQUESTED** [review]
looks good - just need to update requirements/dev-mac-requirements.txt

File: requirements/build-requirements.txt [hunk 1]
```diff
@@ -8,8 +8,8 @@ markupsafe==1.1.1 --hash=sha256:1733a3715d37d075d79b6fe1c2e1055e8cf471fd7da53cdd
 pathlib2==2.3.2 --hash=sha256:460e67b14d0574b0529a0017b1eb05d10d9722681e303fec7077c2a628de60c1
 python-dateutil==2.7.5 --hash=sha256:f6eb9c17acd5a6954e1a5f2f999a41de3e7e25b6bc41baf6344bd053ec25ceeb
 python-editor==1.0.3 --hash=sha256:e47dcec4ea883853b8196fbd425b875d7ec791d4ede2e20cfc70b9a25365c65b
-requests==2.20.0 --hash=sha256:d87b2085783d31d874ac7bc62660e287932aaee7059e80b41b76462eb18d35cc
+requests==2.22.0 --hash=sha256:e3711bd465fd013abe428a8dade2938b578f05d2b06b0ae83daa98869c4548e8
```
[the remark this answers is not in the document]
Comment (zenmonkeykstop) [reply requirements/build-requirements.txt:11]: OK, macOS build reqs updated.

**Review by creviera — APPROVED** [review]
lgtm!

**Pull request merged**